### PR TITLE
Initialize the project

### LINKED ISSUE #1: Initiate the project
**Is your feature request related to a problem? Please describe.**
While we are having discussions on this project we need to create a basic structure for the project.

**Describe the solution you'd like**
- Initiate a maven project with two modules for both frontend and backend. 
- Initiate a SpringBoot project on the backend module
- Add a basic user control with spring security

### PR DESCRIPTION
**Pull request by YohanAvishke**
## Purpose

The purpose of this PR is to fix #1 

## Goals
Init project with required dependencies

## Approach
Initiated a project using Spring Boot Initializr and added other needed dependencies

### Screenshots
N/A
  
### Preview Link
N/A

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

## Related PRs
N/A

## Test environment
 
macOS 10.15, JDK 1.8

## Learning

https://medium.com/@yaediriweera/spring-boot-hello-world-using-spring-initializr-11bd6ee288f7

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+### NetBeans ###
+/nbproject/private/
+/nbbuild/
+/dist/
+/nbdist/
+/.nb-gradle/
+build/
+
+### VS Code ###
+.vscode/
```
Comment (jayasanka-sack) [inline .gitignore:31]:
```suggestion
.vscode/

## macOS ##
.DS_Store
```

File: README.md [hunk 2]
```diff
@@ -1 +1,6 @@
-# scholarx
\ No newline at end of file
+# scholarx
+
+###TODO
+1. Learn about pom resources
+2. Update Application.properties file
+3. Add missing pom resources
```
Comment (jayasanka-sack) [inline README.md:6]: What do you mean by this?

Comment (YohanAvishke) [reply README.md]: Removed

File: .gitignore (hunk 1 above)
Comment (YohanAvishke) [reply .gitignore:31]: Added the missing line

**Pull request merged**